packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -382,3 +382,43 @@
         }
     }
 }
+
+.header-v2 {
+    display: flex;
+    justify-content: center;
+
+    &__acc-info {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+
+        &--top {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            gap: 0.4rem;
```

Comment:
can we use quill token here?

Reply:
i think we are waiting for prince to turn quill-tokens into a library. So that i can put quill-token and package.json and use the tokens